aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -593,6 +606,11 @@ module aptos_framework::account {
         let new_auth_key = from_bcs::to_address(new_auth_key_vector);
         table::add(address_map, new_auth_key, originating_addr);
 
+        event::emit<KeyRotation>(KeyRotation {
```

Comment:
<KeyRotation> is not necessary

Reply:
why?

Reply:
< KeyRotation >

Reply:
> why?

It's because the type can be inferred from the type of the message.